docs/api/wasm/extensions.md

```diff
@@ -3,74 +3,76 @@ layout: docu
+As with regular DuckDB extensions, DuckDB-Wasm extension are by default checked on `LOAD` to verify the signature confirm the extension has not been tampered with.
 Extension signature verification can be disabled via a configuration option.
-Signing is a property of the binary itself, so copying a DuckDB extension (say to serve it from a different location) will still keep a valid signature (for example for local development).
+Signing is a property of the binary itself, so copying a DuckDB extension (say to serve it from a different location) will still keep a valid signature (e.g., for local development).
 
-### Fetching DuckDB-Wasm extensions
+### Fetching DuckDB-Wasm Extensions
 
-DuckDB official extension are served at extensions.duckdb.org, and this is also the default value for the `default_extension_repository` option.
-On installing extensions, a relevant URL will be built that will look like `extensions.duckdb.org/$duckdb_version_hash/$duckdb_platform/$name.duckdb_extension.gz`.
+Official DuckDB extensions are served at `extensions.duckdb.org`, and this is also the default value for the `default_extension_repository` option.
+When installing extensions, a relevant URL will be built that will look like `extensions.duckdb.org/$duckdb_version_hash/$duckdb_platform/$name.duckdb_extension.gz`.
 
 DuckDB-Wasm extension are fetched only on load, and the URL will look like: `extensions.duckdb.org/duckdb-wasm/$duckdb_version_hash/$duckdb_platform/$name.duckdb_extension.wasm`.
 
 Note that an additional `duckdb-wasm` is added to the folder structure, and the file is served as a `.wasm` file.
 
-DuckDB-Wasm extension are served pre-compressed using brotli compression. While fetched from a browser, extensions will be transparently uncompressed. If you want to fetch duckdb-wasm extension manually, you can use `curl --compress extensions.duckdb.org/......../icu.duckdb_extension.wasm`.
+DuckDB-Wasm extensions are served pre-compressed using Brotli compression. While fetched from a browser, extensions will be transparently uncompressed. If you want to fetch the `duckdb-wasm` extension manually, you can use `curl --compress extensions.duckdb.org/<...>/icu.duckdb_extension.wasm`.
 
-### Serving extension from a third party repository
+### Serving Extensions from a Third-Party Repository
 
 As with regular DuckDB, if you use `SET custom_extension_repository = some.url.com`, subsequent loads will be attempted at `some.url.com/duckdb-wasm/$duckdb_version_hash/$duckdb_platform/$name.duckdb_extension.wasm`.
 
-Note that GET requests on the extensions needs to be CORS enabled for a browser to allow the connection.
+Note that GET requests on the extensions needs to be [CORS enabled](https://www.w3.org/wiki/CORS_Enabled) for a browser to allow the connection.
 
 ### Tooling
 
-Both extensions and the deployed DuckDB have been compiled using Emscripten 3.1.45.
+Both DuckDB-Wasm and its extensions have been compiled using Emscripten 3.1.45.
```

Comment:
```suggestion
Both DuckDB-Wasm and its extensions have been compiled using latest packaged Emscripten toolchain.
```

Reply:
Maybe it's better to avoid the precise number, given it's already bumped to 3.1.46 and I don't think we want to update it